formats/gltf/model.go

```diff
@@ -26,6 +26,7 @@ type PolyformModel struct {
 type PolyformMaterial struct {
 	Name                 string
 	Extras               map[string]any
+	AlphaMode            MaterialAlphaMode
```

Comment:
Let's go ahead and make this a pointer
```suggestion
	AlphaMode            *MaterialAlphaMode
```